src/devices/BrickPi3/Models/Sensors.cs

```diff
@@ -71,40 +107,42 @@ public enum SensorType : byte
     /// </summary>
     public enum SensorState : byte
     {
+        /// <summary>Valid data</summary>
         ValidData = 0,
+        /// <summary>Not configured</summary>
         NotConfigured,
+        /// <summary>Configuring</summary>
         Configuring,
+        /// <summary>No data</summary>
         NoData,
+        /// <summary>I2C error</summary>
         I2CError
     }
 
     /// <summary>
-    ///     Flags for use with SENSOR_TYPE.CUSTOM
-    /// PIN1_9V
-    ///     Enable 9V out on pin 1 (for LEGO NXT Ultrasonic sensor).
-    /// PIN5_OUT
-    ///     Set pin 5 state to output.Pin 5 will be set to input if this flag is not set.
-    /// PIN5_STATE
-    ///    If PIN5_OUT is set, this will set the state to output high, otherwise the state will
-    ///    be output low.If PIN5_OUT is not set, this flag has no effect.
-    /// PIN6_OUT
-    ///    Set pin 6 state to output.Pin 6 will be set to input if this flag is not set.
-    /// PIN6_STATE
-    ///    If PIN6_OUT is set, this will set the state to output high, otherwise the state will
-    ///    be output low.If PIN6_OUT is not set, this flag has no effect.
-    /// PIN1_ADC
-    ///    Enable the analog/digital converter on pin 1 (e.g. for NXT analog sensors).
-    /// PIN6_ADC
-    ///     Enable the analog/digital converter on pin 6.
+    /// Flags for use with SENSOR_TYPE.CUSTOM
     /// </summary>
     public enum SensorCustom
     {
+        /// <summary>Enable 9V out on pin 1 (for LEGO NXT Ultrasonic sensor).</summary>
         Pin1_9V = 0x0002,
+        /// <summary>Set pin 5 state to output.Pin 5 will be set to input if this flag is not set.</summary>
         Pin5_Out = 0x0010,
+        /// <summary>
+        /// If PIN5_OUT is set, this will set the state to output high, otherwise the state will
+        /// be output low.If PIN5_OUT is not set, this flag has no effect.
+        /// </summary>
         Pin5_State = 0x0020,
+        /// <summary>Set pin 6 state to output.Pin 6 will be set to input if this flag is not set.</summary>
         Pin6_Out = 0x0100,
+        /// <summary>
+        /// If PIN6_OUT is set, this will set the state to output high, otherwise the state will
```

Comment:
Do we have a rule in our coding guidelines about adding endlines in the middle of sentences? I ask because when we port these multi-line comments, they will have to be manually merged into a single line (unless there is a dot or a colon).

Reply:
I'm generally fine with that as part of this PR. I suspect in the future we can modify the tool to be more noisy and report all of these